tests/BasicSearchTests.FunctionalTests.Core/V3SearchFunctionalTests.cs

```diff
@@ -37,5 +37,23 @@ public class V3SearchFunctionalTests : BaseFunctionalTests
             Assert.True(result.TotalHits.HasValue && result.TotalHits.Value > 0, "No results found, should find atleast some results for empty string query.");
             Assert.NotNull(result.Data);
         }
+
+        [Fact]
+        public async Task ResultForJsonNetHasDownloads()
+        {
+            // Act
+            var response = await Client.GetAsync(new V3SearchBuilder() { Query = "JSON.Net" }.RequestUri);
+            var result = await response.Content.ReadAsAsync<V3SearchResult>();
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.True(result.TotalHits.HasValue && result.TotalHits.Value > 0, "No results found, should find atleast some results for empty string query.");
```

Comment:
String is descriptive for both cases, it doesn't matter if they are identical or not.
The assertions should NOT be moved because as they are, they are very explicit about what is being verified here. There is no guarantee that these being the same will always be the case, and looking at each test should be a self contained package of what is being tested and what conditions are being checked.